manifests/squid.pp

```diff
@@ -0,0 +1,55 @@
+# The class to manage squid. This is used by pulp streamer.
+class pulp::squid(
+  Integer[0, 65535] $port = 3128,
```

Comment:
I'm guessing this PR might have predated `Stdlib::Port`?

Reply:
At least predates me being aware/using it.